docs/resources/check_group.md

```diff
@@ -82,8 +92,26 @@ resource "checkly_check_group" "test_group1" {
 
 # Add a check to a group
 resource "checkly_check" "test_check1" {
-  name                      = "My test check 1"
```

Comment:
This check is missing some required properties (probably just to keep things concise). To make things easier for users copying and pasting from the examples, though, I think we can add the extra properties here.